db/seeds/05_blocked_usernames.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+%w(
+  abuse
+  account
+  accounts
+  admin
+  administration
+  administrator
+  admins
+  help
+  helpdesk
+  instance
+  mod
+  moderator
+  moderators
+  mods
+  owner
+  root
+  security
+  server
+  staff
+  support
+  webmaster
+).each do |str|
+  UsernameBlock.create_with(username: str, exact: true).find_or_create_by(username: str)
+end
+
+%w(
+  mastodon
+  mastadon
+).each do |str|
+  UsernameBlock.create_with(username: str, exact: false).find_or_create_by(username: str)
+end
```

Comment:
I would perhaps allow with approval here, we have a few accounts of our own with “Mastodon” in the username, and there are accounts around such as “mastodonusercount” or similar. Also any hope of having the Mastodon band :D